plugins/hydra_ax_sweeper/tests/test_ax_sweeper_plugin.py

```diff
@@ -37,6 +37,15 @@ def quadratic(cfg: DictConfig) -> Any:
     return z
 
 
+def nested_quadratic_with_escape_char(cfg: DictConfig) -> Any:
+    x = cfg.quadratic.x_arg
+    y = cfg.quadratic.y_arg
+    a = 100
+    b = 1
+    z = a * (x ** 2) + b * y
+    return z
```

Comment:
This this is a test, there is no point in being so verbose (and same for other tests)
```suggestion
    return 100 * (cfg.quadratic.x_arg ** 2) + 1 * cfg.quadratic.y_arg
```